react/ProductDescription.tsx

```diff
@@ -55,14 +66,31 @@ function ProductDescription(props: PropsWithChildren<Props>) {
   const { handles } = useCssHandles(CSS_HANDLES, { classes: props.classes })
   const intl = useIntl()
   const { product } = useProduct()
+  const { isMobile } = useDevice()
 
   const description = props.description ?? product?.description
 
   if (!description) {
     return null
   }
+  
+  const { collapseContent = true, title, customCollapse = false } = props
```

Comment:
Also, I think you're changing the default behavior here, since collapseContent used to default to false. That's a breaking change.

Reply:
Yes. If you use customCollapse you will overwrite the default collapseContent behavior.

Reply:
Oh, I see! I didn't think `collapseContent` defaulted to `true`, my bad 😅